init.ts

```diff
@@ -54,7 +56,7 @@ async function createSuperuser() {
 	const { ConflictError } = sbvrUtils;
 
 	const data = {
-		username: 'root',
+		username: SUPERUSER_USERNAME,
```

Comment:
This will create a new user every time the username changes. I would hardcode `'admin'` right here, like we had `'root'` before.

Reply:
Nice catch, will update.